.circleci/config.yml

```diff
@@ -74,13 +74,13 @@ jobs:
       - run:
           name: 🚧 install dependencies from latest master commit
           command: |
-              git clone --depth 1 https://github.com/plotly/dash-core-components.git
-              git clone --depth 1 https://github.com/plotly/dash-html-components.git
+              git clone https://github.com/plotly/dash-core-components.git
+              git clone https://github.com/plotly/dash-html-components.git
               git clone --depth 1 https://github.com/plotly/dash-table.git
               git clone --depth 1 https://github.com/plotly/dash-renderer-test-components
               . venv/bin/activate
-              cd dash-core-components && npm install --ignore-scripts && npm run build && pip install -e . && cd ..
-              cd dash-html-components && npm install --ignore-scripts && npm run build && pip install -e . && cd ..
+              cd dash-core-components && git checkout 2932409 && npm install --ignore-scripts && npm run build && pip install -e . && cd ..
```

Comment:
after these two commits, dev bundle was removed in dcc and html, which was the root cause of the 4 devtools props-check related failure. 

we need to address that with the formal release and this is a workaround for this PR to pass all tests